.github/CODEOWNERS

```diff
@@ -1,8 +1,13 @@
-#Admins & project code owners
-#*           @rapidsai/admins @rapidsai/cudf-admin @rapidsai/cudf-codeowners
-
 #cpp code owners
-#cpp/       @rapidsai/cudf-cpp-codeowners
+cpp/       @rapidsai/cudf-cpp-codeowners
 
 #python code owners
-#python/    @rapidsai/cudf-python-codeowners
+python/    @rapidsai/cudf-python-codeowners
+
+#build/ops code owners
+.github/           @rapidsai/cudf-ops-codeowners 
+ci/                @rapidsai/cudf-ops-codeowners
+conda/             @rapidsai/cudf-ops-codeowners
+docker/            @rapidsai/cudf-ops-codeowners
+**/CMakeLists.txt  @rapidsai/cudf-ops-codeowners
```

Comment:
I'm unfamiliar with this syntax of `**/`. Does this match against any file named `CMakeLists.txt`? 

If so, I think it's going to be too annoying to require an ops review on any modification to `cpp/CMakeLists.txt`.

Reply:
Agree, any C++ owner should be able to approve any cmake change, it shouldn't require ops.

Reply:
This was at the request of @kkraus14.

Reply:
> I'm unfamiliar with this syntax of `**/`. Does this match against any file named `CMakeLists.txt`?
> 
Yes, matches that filename in any directory in the entire repo.

Reply:
It's going to be very annoying if we _require_ a review an approving review from cudf-ops-codeowners for any CMake change.

Anytime a new feature that merits a new file is added (or a new GTest file is added) it requires updating the CMakeLists.txt. This happens pretty frequently, and I think it's going to be vexing (for all of us) if that always requires a cudf-ops-codeowners approval.

Reply:
@jrhemstad how about a cudf-cmake-codeowners?

Reply:
@raydouglass Looks like we've got agreement above.

Reply:
> @jrhemstad how about a cudf-cmake-codeowners?

I think that's just going to be union of the present members of `cudf-ops-codeowners` and `cudf-cpp-codeowners`, but I suppose it's worth making it's own group in case we ever want to differentiate.

Reply:
In general, cpp code owners will always need to be cmake codeowners.